.github/workflows/docker.yml

```
@@ -9,6 +9,7 @@ on:
     push:
         branches:
             - main
+            - support/*
```

Think this should be different (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-filters)
```suggestion
            - 'support/**'
```

---

Yes, I guess so. Thanks for the correction.